### PR TITLE
Fix metrics server

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*
* Fixed logic due to metric server version change

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: development/kops/cluster_wait.sh [hunk 1]
```diff
@@ -71,4 +68,4 @@ k -n kube-system delete pods -lk8s-app=cilium
 set -x
 ${KOPS} validate cluster --wait 15m
 
-# --set ipam.mode="kubernetes" 
\ No newline at end of file
+# --set ipam.mode="kubernetes"
```
Comment (kschumy) [inline development/kops/cluster_wait.sh:71]: IntelliJ automatically added an empty line, and I opted to keep this change.

File: development/kops/cluster_wait.sh [hunk 2]
```diff
@@ -42,21 +42,18 @@ function k {
...
-# In kops 1-22 metrics was updated and the port was changed to 443 from 4443. In the verison of metrics server we ship for kube versions < 1-22, it does not support binding to 443
-# patching back to old port and behavior
-if [ "${RELEASE_BRANCH}" != "1-22" ]; then
+# TODO: Delete when discontinue support for 1-18
+if [ "${RELEASE_BRANCH}" == "1-18" ]; then
     PATCH='[{"op": "add", "path": "/spec/template/spec/containers/0/args/-", "value": "--secure-port=4443" },{"op": "replace", "path": "/spec/template/spec/containers/0/ports/0/containerPort", "value": 4443 }]'
     k  -n kube-system patch deployments metrics-server --type=json -p="$PATCH"    
 fi
 
-# kops 1-22 installs an older metrics server than we ship with eksd 1.22 along with an older clusterrole def.  The 0.6 version of metrics requires a slightly different rbac setup
```
Comment (jaxesn) [inline development/kops/cluster_wait.sh:57]: We might want to leave this comment as a clue for the next time we update kops to revisit this.

**Review by jaxesn — CHANGES_REQUESTED** [review]

File: development/kops/cluster_wait.sh (hunk 2 above)
Comment (kschumy) [reply development/kops/cluster_wait.sh:57]: Added back and updated to fit the change proposed in this PR

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**